docs/CHANGELOG.rst

```diff
@@ -32,6 +32,10 @@ Added
 * Add ``prepare-geo-chipseq`` and ``prepare-geo-rnaseq`` processes that
   produce a tarball with necessary data and folder structure for GEO
   upload
+* BaseSpace file download tool
```

Comment:
Use _Add_ at the beginning of all four added change log entries, like in the rest of this section.